package.json

```diff
@@ -66,6 +66,6 @@
     "typescript": "^4.5.5"
   },
   "engines": {
-    "node": "^12 || ^14 || ^16 || ^17"
+    "node": "^12 || ^14 || ^16 || ^17" || ^18
```

Comment:
oh my mistake...
Must be written in double quotes...
I hope the intention is conveyed.

Reply:
https://github.com/tyankatsu0105/cz-format-extension/pull/109/commits/6da47309501b1f9b0ff5e34b54e2491116aefa6c

done!